task/01-strings-tasks.js

```diff
@@ -237,8 +268,11 @@ function encodeToRot13(str) {
  *   isString('test') => true
  *   isString(new String('test')) => true
  */
-function isString(value) {
-    throw new Error('Not implemented');
+function isString(value) {         
+    
+  return typeof(value) == 'string' ? true :
+         (typeof(value) == 'undefined' || value == null) ? false :
+         (typeof(value.valueOf())) == 'string' ? true : false;
```

Comment:
avoid this code style, if-else is more readable than multiple conditional operators `? :`